app/assets/javascripts/admin/addon/templates/users-list-show.hbs

```diff
@@ -240,11 +240,16 @@
             {{/if}}
 
             <div
-              class="directory-table__cell user-role{{if
-                  (or user.admin user.moderator user.second_factor_enabled)
-                  ''
-                  '--empty'
-                }}"
+              class={{concat-class
+                "directory-table__cell"
+                "user-role"
+                (if
+                  (not
+                    (or user.admin user.moderator user.second_factor_enabled)
```

Comment:
I have a hard time reading "if ... not ... or". Since this div is getting `display:none` when it has the `--empty` class, would it make sense to move the if statement higher up the DOM and only output `div.user-role` when necessary?